### PR TITLE
Creates merge_studio_persons script.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: bin/oneoff/merge_studio_persons [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+
+# This script reads a CSV of data, connecting users via studio_persons using the
+# email address links in the CSV. The CSV is expected to contain at least two
+# columns (later columns are ignored) of emails and alternate emails.
+
+require_relative '../../dashboard/config/environment'
+require 'csv'
+
+FILENAME = ARGV[0].freeze
```
Comment (aoby) [inline bin/oneoff/merge_studio_persons:10]: Do we expect this script to be used in an ongoing basis, or just one batch? If the former, it should be moved outside of `oneoff`.

File: bin/oneoff/merge_studio_persons [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  email = line[0].downcase.strip
+  alternate_email = line[1].downcase.strip
+  data << [email, alternate_email]
+end
+
+data.each do |email, alternate_email|
+  # Try to add alternate_email to the StudioPerson associated with email.
+  user = User.find_by_email_or_hashed_email(email)
+  if user
+    studio_person = user.studio_person
```
Comment (aoby) [inline bin/oneoff/merge_studio_persons:23]: Are they guaranteed to have a `studio_person`? Can it be nil?

File: bin/oneoff/merge_studio_persons [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    studio_person.add_email(alternate_email)
+    next
+  end
+
+  # Otherwise, try to add email to the StudioPerson associated with alternate_email.
+  user = User.find_by_email_or_hashed_email(alternate_email)
+  if user
+    studio_person = user.studio_person if user
```
Comment (aoby) [inline bin/oneoff/merge_studio_persons:31]: same as above

File: bin/oneoff/merge_studio_persons (hunk 1 above)
Comment (ashercodeorg) [reply bin/oneoff/merge_studio_persons:10]: Mostly oneoff, its location seems good.

File: bin/oneoff/merge_studio_persons (hunk 2 above)
Comment (ashercodeorg) [reply bin/oneoff/merge_studio_persons:23]: Yes. No.

File: bin/oneoff/merge_studio_persons (hunk 3 above)
Comment (ashercodeorg) [reply bin/oneoff/merge_studio_persons:31]: Same.

**Pull request merged**